spec/graphql/relay/page_info_spec.rb

```diff
@@ -75,8 +75,8 @@ def get_last_cursor(result)
       result = star_wars_query(query_string, "last" => 1, "first" => 1, "before" => cursor_of_last_base)
       assert_equal(true, get_page_info(result)["hasNextPage"])
       assert_equal(true, get_page_info(result)["hasPreviousPage"])
-      assert_equal("Mg==", get_page_info(result)["startCursor"])
-      assert_equal("Mg==", get_page_info(result)["endCursor"])
+      assert_equal("MQ==", get_page_info(result)["startCursor"])
+      assert_equal("MQ==", get_page_info(result)["endCursor"])
```

Comment:
If I remember correctly the `star_wars_query` here without any of `first`/`last`/`before`/`after` will return 3 bases: `Death Star`, `Shield Generator` and `Headquarters` (in this order).

The `cursor_of_last_base` points to `Headquarters`, so without `first`/`last` the result would contain `Death Star` and `Shield Generator`.

According to the spec, `first`/`last` have to be applied one after the other. In this case here, applying `first: 1` first reduces the result set to `Death Star` only (cursor value: `MQ==`), and applying `last: 1` does not change the result.

The previous code would return ``

Previously, the cursor for `Shield Generator` was returned, which did not match the spec.